cmd/scale.go

```diff
@@ -205,11 +205,8 @@ func (sc *scaleCmd) load(cmd *cobra.Command) error {
 		return errors.Wrap(err, "error while trying to unmarshal the ARM template")
 	}
 
-	templateMap := template.(map[string]interface{})
```

Comment:
Do we still need template for anything else? If not, can the lines above (starting at 193) be removed?

Reply:
Also, this is a stretch but - can we remove the required "deployment dir" arg with this change and replace it by an "apiModelPath" since that's the only file we should need now?

Reply:
I don't believe we need it anywhere. I'll remove these lines as well.